src/main/resources/case_handed_offline.bpmn

```diff
@@ -52,50 +52,61 @@
     <bpmn:sequenceFlow id="Flow_1sfazbk" sourceRef="StartEvent_1" targetRef="Activity_14l4zql" />
     <bpmn:sequenceFlow id="Flow_07i7aul" sourceRef="Activity_14l4zql" targetRef="CaseHandedOfflineDefendantTaskId" />
     <bpmn:sequenceFlow id="Flow_0x8zuet" sourceRef="Activity_14l4zql" targetRef="CaseHandedOfflineClaimantTaskId" />
+    <bpmn:callActivity id="Activity_0muk2lk" name="End Business Process" calledElement="EndBusinessProcess">
+      <bpmn:extensionElements>
```

Comment:
<img width="253" alt="Screenshot 2020-10-15 at 12 34 59" src="https://user-images.githubusercontent.com/58216515/96118140-0047c280-0ee3-11eb-9a73-ae1a34f6796b.png">

That sneaky line can cause some weird problems. 😄 Same on claim issue diagram.

Reply:
good spot!!